vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc

```diff
@@ -1866,8 +1866,33 @@ bool getJSONValue(const std::string& fieldName,
 
   // has parser errors or wrong types
   bool error = d.HasParseError() || (false == d.HasMember(fieldName.c_str()));
+
+  // ensure string type
+  rapidjson::Value* item = &d[fieldName.c_str()];
+  error = (item->GetType() == rapidjson::Type::kStringType) && error;
```

Comment:
Added  type check here to avoid error. Probably needs refactor `getJSONValue` -> `getJSONStringValue`